homework/index.js

```diff
@@ -30,14 +30,93 @@
     return elem;
   }
 
+  function createOption(repositories, select) {
+    createAndAppend('option', select, { text: 'Select a repository', disabled: 'disabled' });
+    repositories
+      .sort((a, b) => a.name.localeCompare(b.name))
+      .forEach(repository => {
+        createAndAppend('option', select, { text: repository.name, value: repository.name });
+      });
+  }
+
+  function renderRepoBasisInfo(repository, ul) {
+    const url = `https://api.github.com/repos/HackYourFuture/${repository}`;
```

Comment:
There is no need to fetch the repo details again. We already have this data (for all repos), fetched in `main()`. You pass the selected repository object as a parameter to this function rather than fetching it again.